CMakeLists.txt

```diff
@@ -164,6 +164,53 @@ function (monero_add_minimal_executable name)
     monero_set_target_no_relink( ${name} )
 endfunction()
 
+function(forbid_undefined_symbols)
```

Comment:
I would love to see about 2 lines of documentation what this function does and how, as well as some detailed but shorter explanation of what goes on within the function.

Reply:
Added a comment.